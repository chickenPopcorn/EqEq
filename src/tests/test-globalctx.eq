myVal = 42; /* in the "global" context */

/* `find` doesn't prefix with a "context" */
find myVal {
  print("myVal is defined on the global context as %.0f\n", myVal);
}
