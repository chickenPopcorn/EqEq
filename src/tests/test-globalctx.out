myVal is defined on the global context as 42
